boards/arm/stm32h747i_disco/stm32h747i_disco_m7.dts

```diff
@@ -21,6 +21,12 @@
 		zephyr,flash = &flash0;
 	};
 
+	sdram2: sdram@d0000000 {
+		/* FIXME: replace compatible (not SRAM) */
+		compatible = "mmio-sram";
```

Comment:
This should be device_type = "memory" instead of compatible="mmio-sram"

Reply:
See example in boards/arm/mimxrt1060_evk/mimxrt1060_evk.dts

Reply:
fixed, thanks